src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,113 +59,267 @@ const paddingsBottom = {
     large: "tw-pb-4",
 };
 
-const TooltipComponent: ForwardRefRenderFunction<HTMLDivElement, TooltipProps> = (
-    {
-        content,
-        tooltipIcon,
-        heading,
-        headingIcon,
-        linkUrl,
-        linkLabel,
-        brightHeader,
-        buttons,
-        tooltipAriaProps = {},
-        style,
-        children,
-        popperAttributes,
-    },
-    tooltipContainerRef,
-) => {
+export enum TooltipPosition {
+    Top = "Top",
+    Right = "Right",
+    Bottom = "Bottom",
+    Left = "Left",
+}
+
+export enum TooltipAlignment {
+    Start = "Start",
+    Middle = "Middle",
+    End = "End",
+}
+
+const placementMap: Record<`${TooltipPosition}-${TooltipAlignment}`, Placement> = {
+    ["Top-Start"]: "top-start",
+    ["Top-End"]: "top-end",
+    ["Bottom-Start"]: "bottom-start",
+    ["Bottom-End"]: "bottom-end",
+    ["Left-Start"]: "left-start",
+    ["Left-End"]: "left-end",
+    ["Right-Start"]: "right-start",
+    ["Right-End"]: "right-end",
+    ["Top-Middle"]: "top",
+    ["Right-Middle"]: "right",
+    ["Bottom-Middle"]: "bottom",
+    ["Left-Middle"]: "left",
+};
+
+const getArrowClasses = (currentPlacement: string, brightHeader: BrightHeaderStyle | undefined, alignment: string) => {
+    switch (true) {
+        case currentPlacement.toString().includes("top"):
```

Comment:
Isn't it 
```suggestion
        case currentPlacement.toString().includes(TooltipPosition.Top):
```

Reply:
It's TooltipPosition.Top.toLowerCase(), Do we want it like this?

Reply:
I read the code a bit more, now I see why 👍